README.md

```diff
@@ -8,39 +8,25 @@ See https://curl.haxx.se/docs/manpage.html or in your shell use `curl --help` fo
 
 ## Usage
 
-Include the `highlight.js` script package in your webpage or node app, load this module and register it with `hljs`. Follow instructions at [highlightjs](https://highlightjs.org/) to learn how to include the library and CSS.
+Include the `highlight.js` script package in your webpage or node app, load this module and register it with `hljs`. Follow instructions at [highlightjs](https://highlightjs.org/) to learn how to include the library and CSS. See [Getting started](https://github.com/highlightjs/highlight.js#getting-started) for different integration and module options.
```

Comment:
That's an interesting choice I didn't see coming... do you think this is better than including your own example of usage (which you've removed below)?